luaui/Widgets/map_startbox.lua

```diff
@@ -213,6 +213,214 @@ local function drawFormList()
 	end
 end
 
+---------------------------------- StartPolygons ----------------------------------
+-- Note: this is now updated to support arbitrary start polygons via GL.SHADER_STORAGE_BUFFER
+
+-- The format of the buffer is the following:
+-- Triplets of :teamID, triangleID, x, z
+
+-- Spring.Echo(Spring.GetTeamInfo(Spring.GetMyTeamID()))
+
+-- TODO:
+-- [ ] Handle overlapping of boxes and myAllyTeamID
+-- [X] Handle Minimap drawing too 
+	-- [X] handle flipped minimaps 
+-- [ ] Pass in my team too
+-- [ ] Handle Scavengers in scavenger color
+-- [ ] Handle Raptors in raptor color
+
+local scavengerStartBoxTexture = "LuaUI/Images/scav-tileable_v002_small.tga"
+
+local raptorStartBoxTexture = "LuaUI/Images/rapt-tileable_v002_small.tga"
+
+local getMiniMapFlipped = VFS.Include("luaui/Widgets/Include/minimap_utils.lua").getMiniMapFlipped
+
+local scavengerAITeamID = 999
+local raptorsAITeamID = 999
+local scavengerAIAllyTeamID
+local raptorsAIAllyTeamID
```

Comment:
Do the teamIDs need the magic 999 initialisation? There doesn't seem to be any arithmetic done on them
```suggestion
local scavengerAITeamID
local raptorsAITeamID
local scavengerAIAllyTeamID
local raptorsAIAllyTeamID
```